oak_functions/examples/hello_world/module/src/lib.rs

```diff
@@ -0,0 +1,67 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+//! Oak functions hello world example.
+//!
+//! To build and run manually:
+//! 1. Build the wasm module:
```

Comment:
In order to be valid markdown, there should be newlines before paragraphs and code blocks, and also the code blocks should be indented if they are part of the bullet point lists.

Reply:
Thanks. I added newlines and indentation. Does it look like a valid markdown now?

Reply:
I think it's still missing the newlines before each code block, and also the code blocks themselves need to be indented (not the code inside the blocks). Does it render correctly currently? BTW I think this would be more maintainable and readable as an actual .md file in the directory, rather than here, since we don't tend to use the generated Rust documentation much, WDYT?